pypose/lietensor/basics.py

```diff
@@ -122,6 +122,86 @@ def add(input, other, alpha=1):
     '''
     return input.add(other, alpha)
 
+def mul(input, other):
+    r'''
+    Multiplies input LieTensor by other.
+
+    .. math::
+        \bm{y}_i =
+        \bm{x}_i \times \bm{a}_i
+
+    where :math:`\bm{x}` is the ``input`` LieTensor, :math:`\bm{a}` is the ``other`` value,
+    and :math:`\bm{y}` is the output value.
+    
+    :obj:`LieTensor` @ :obj:`LieTensor` -> :obj:`LieTensor` (group multiplication) 
+        
+    :obj:`LieTensor` @ :obj:`Tensor` -> :obj:`Tensor` (group action)
+
+    Args:
+        input (:obj:`LieTensor`) the input LieTensor (Lie Group or Lie Algebra).
+
+        other (:obj:`Tensor` or :obj:`LieTensor`): the value for input to be multiplied by.
+
+    Return:
+        the product with the same type as ``other``
```

Comment:
You need a type for returns. For example,
```
Tensor/LieTensor: the product with the same type as ``other``.
```
Please refer to other docs for function return.